src/configs/mod.rs

```diff
@@ -451,6 +455,33 @@ impl WireguardNetworkInfo {
     pub fn has_flag(&self, flag_name: &str) -> bool {
         self.flags.iter().any(|f| f.as_ref() == flag_name)
     }
+
+    pub fn get_free_net_address(&self, net: IpNetwork, num: u128) -> IpAddr {
+        let mut cum = 0u128;
```

Comment:
sorry, what?

Reply:
cumulative sum. Ok I'll rename it somehow